src/helpers/dockerCheck.js

```diff
@@ -76,4 +78,21 @@ module.exports = class DockerCheck {
       });
     });
   }
+
+  static async checkDockerComposeVersion() {
+    console.log('Checking docker compose version...')
+    //We are executing both commands because in Linux we may have docker-compose v2, so we need to check both
+    const resultFirstCommand = await shell.exec(`docker compose version --short`, {silent:true});
+    const resultSecondCommand = await shell.exec(`docker-compose version --short`, {silent:true})
+
+    // Exit code is 127 when no docker installation is found
+    if (resultFirstCommand.code === 127 && resultSecondCommand.code === 127) {
+        console.error('Seems like you do not have docker compose installed');
+    } else {
+      const version = resultFirstCommand.stdout ? resultFirstCommand.stdout : resultSecondCommand.stdout
+      if (semver.lt(version, '2.12.2')) {
+        console.error("You are using docker compose version prior to 2.12.2, please upgrade");
+      }
+    }
+  }
```

Comment:
IMO, inversing the logic is the better approach in that case. We have requirements that the end-user has to satisfy. We don't care if is it using an old version of `docker compose` or didn't have `docker compose` at all.
```typescript
  static async checkDockerComposeVersion() {
    console.log('Checking docker compose version...');
    
    const dockerComposeVersion = await shell.exec(`docker compose version --short`, {silent:true});
    if (dockerComposeVersion.code === "<success_code>" && semver.gte(dockerComposeVersion?.stdout, '2.12.2')) {
      // seems good
      return true;
    }

    console.error('The hedera-local-node package requires docker compose >= 2.12.2');
  }
  ```

That's not a change request, just my thoughts. Aside from that, the PR LGTM :upside_down_face:  Approved. :confetti_ball: